src/proto/diff.proto

```diff
@@ -24,23 +24,23 @@ message ChangedFile {
 		/** The file represents a Java source file that had a parse error */
 		SOURCE_JAVA_ERROR = 100;
 		/** The file represents a Java source file that parsed without error as JLS2 */
-		SOURCE_JAVA_JLS2 = 101;
```

Comment:
any reason to skip 101?

Reply:
so that JLSi is mapped to 10i.

I also thought about adding JLS1=101 but then I thought it's not important nor useful. What do you think?

Reply:
I mean I guess, but it honestly doesnt matter what proto field number is used for this.  You could map to 184 if you wanted.  So I was just curious why we were changing it at all.

Reply:
I did add JLS. That's when I modified so that JLSi=10i. I think it would be easier to keep track.
Then I removed JLS1.

Reply:
I am ok with this change, I was just wondering the reasoning behind it.  In general we don't have to worry about maintaining the same field numbers as we do not support older datasets in newer compilers.